plugin-server/src/utils/db/db.ts

```diff
@@ -424,13 +424,18 @@ export class DB {
 
     // Person
     REDIS_PERSON_ID_PREFIX = 'person_id'
+    REDIS_PERSON_UUID_PREFIX = 'person_uuid'
```

Comment:
is there a typescript templating magic I could be using to make this more easily editable?